lib/generate-id.js

```diff
@@ -22,11 +22,41 @@ const findId = async ( model, length ) => {
 }
 
 /**
- * Generates a line id and a shabad id, using the database.
+   * Logs the generated id to the console
+   * @param {Number} count the count of id to generate
+   * @param {String} type shabad, line
+   */
+const logGeneratedID = async function ( count, type ) {
```

Comment:
Minor nitpick about the function declaration:
1) It logs multiple IDs, so probably better to use the plural here
2) Keep `ID` as `Id`/follow the camelCase rules for abbreviations too
3) Don't use `function` unless absolutely necessary. It has slightly different semantics to the arrow declarations: `const logGeneratedId = async (count, type) => {`
4) You're using the past-tense in the function name, which would indicate that the IDs are already generated when you call the function. I'd personally rename it to `generateIds`.